### DISCUSSION TITLE
Control who recieve AIO notifications (like daily backup successful / failed)

### DISCUSSION THREAD

**Discussion by ecker00** [nextcloud/all-in-one#1740] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 
- Control who receives daily notification that backup was successful / failed. This notification can get quite bothersome when you think there is something important to checkout.
- This seems also relevant (https://github.com/nextcloud/all-in-one/issues/1417), would not want *every* user to receive a notification that the AIO is outdated (I've not seen this notification yet, so not sure how it works, but guess it's similar to the backup?).

### Describe the solution you'd like
It seems AIO notifications are sent to all admins, but not all admins are maintainers, so it's irrelevant information to send to *all* admins. Would be nice to control who/which groups receives what.

### Describe alternatives you've considered
I've tried to move some of our admins to it's own `SubAdmin` group, and give that group administration privileges. But there is no way to give that group right to administer users (/settings/users), so it's not really an admin anymore.

### Additional context
Think it's important to reduce noise for our users, so more important notifications won't be ignored.

**Comment by szaimen** [comment]
Hi, you can already control that youself by adding people to or removing from the admin group.

Also see https://github.com/nextcloud/all-in-one/discussions/1487#discussioncomment-4350012

**Reply by ecker00** [reply]
But that also have the side effect of removing other admin privileges which I can't seem to grant in any other way (like user management).

**Reply by szaimen** [reply]
Privileges for user management of group admins needs to get addressed in server IIRC...

**Reply by EgonHeuson** [reply]
Hello,
I'm not sure to fully understand Simon's answer :-(. I have a similar issue/request. I'd like to only receive the notification only when the backup fails. Do you think this would be feasible? 
Thanks! :-)

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one/discussions/484

**Reply by EgonHeuson** [reply]
Ok I understand the point :-) And if I only want to limit to the main admin but keep the admin privileges for some other Users, I have to create a sub-admin group? Right ? :-)

**Reply by szaimen** [reply, marked as answer]
yes, you would need to use sub-admins then or change the notification settings mentioned in https://github.com/nextcloud/all-in-one/discussions/1487#discussioncomment-4580933

Also see https://github.com/nextcloud/all-in-one/discussions/484

**Reply by EgonHeuson** [reply]
Ok I'll try this! Thanks a lot :-)
